src/app.service.js

```diff
@@ -5,4 +5,25 @@ export class AppService {
   getHello() {
     return "Hello World!";
   }
+
+  getReadingTime(articleBody, userWPM) {
+    const numOfWords = articleBody.split(" ").length;
+    const rawReadingTime = (numOfWords / userWPM) * 60 * 1000;
+    let readingMinute = Math.floor(rawReadingTime / 1000 / 60);
```

Comment:
readingSeconds 와 같이 readingMinutes 로 유사한 형태의 변수명을 사용하는것은 어떨까요?
```suggestion
    let readingMinutes = Math.floor(rawReadingTime / 1000 / 60);
```